_i18n/en.yml

```diff
@@ -4,16 +4,30 @@ global:
     en: English
     es: Español
 meta:
-  description: login.gov, the consumer identity platform
+  description:
+    contact_us: Have a question or a problem with login.gov?
```

Comment:
Question, for organization do you think it's easier to have these all under one `meta`? I think that if I wanted to edit the "privacy" page for example, it would be nice to have `title` and `description` and the content itself all next to each other?